model_inference/bird_predictor.py

```diff
@@ -17,32 +15,49 @@
         6: 'Цыпленок'
     }
 
-def model_upload():
-  output = 'bird_recognition.h5'
-  if not os.path.isfile(output):
-    file_id = '1-bWaePLw8owneHJnK2US7jcYnG5gaI5j'
-    url = f'https://drive.google.com/uc?id={file_id}'
-    gdown.download(url, output, quiet=False)
-  model = load_model(output)
-  return model
+def download_file_from_url(url:str, destination_path:str) -> None:
```

Comment:
Let's try adding docstrings to all classes and functions. Example:
```
def add_binary(a, b):
    '''
    Возвращает сумму двух десятичных чисел в двоичном формате.
            Параметры:
                    a (int): первое десятичное целое число
                    b (int): второе десятичное целое число
            Возвращаемое значение:
                    binary_sum (str): двоичная строка суммы a и b
    '''
    binary_sum = bin(a+b)[2:]
    return binary_sum
```


This is needed so that to all functions or classes you can call the __doc__ method and get docstrings. You can try to paste this example somewhere and call it:
```
print(add_binary.__doc__)
```

Let's do the same thing, but only in English.